includes/pipeline_page/output.php

```diff
@@ -0,0 +1,99 @@
+<?php
+// Build the HTML for a pipeline documentation page.
+// Imported by public_html/pipeline.php - pulls a markdown file from GitHub and renders.
+
+# Details for parsing markdown file, fetched from Github
+# Build the remote file path
+# Special case - root docs is allow
+
+# General docs page
+if($path_parts[1] == 'output'){
+    if(substr($_SERVER['REQUEST_URI'], -3) == '.md'){
```

Comment:
Some slightly variable whitespace going on here 😉 

We should probably set up the editorconfig stuff in this repo as we have [started](https://github.com/nf-core/modules/blob/master/.editorconfig) doing [elsewhere](https://github.com/nf-core/modules/blob/2b6dc71338f206492a0ff00ebaf31214a23d51fe/.github/workflows/lint-code.yml#L20-L33)..